### DISCUSSION TITLE
First Steps....

### DISCUSSION THREAD

**Discussion by Josue267A** [skills/.github#608] [Introduction to GitHub]
Today I learned how to use GITHUB, IT WAS AMAZING!!!!!

**Comment by yashksaini-coder** [comment]
Congratulations on your success. Also best of luck in you journey

**Comment by userpl6789** [comment]
I can't get to the step 2

**Reply by yashksaini-coder** [reply]
Hi @userpl6789 which skill are you learning and what are you currently trying to do in that step ?
